.github/workflows/reusable_track_size.yml

```diff
@@ -91,7 +91,13 @@ jobs:
 
           echo "$data" > "/tmp/data.json"
 
-          comparison=$(python3 scripts/ci/sizes.py compare --threshold=5 "/tmp/prev.json" "/tmp/data.json")
+          comparison=$(
+            python3 scripts/ci/sizes.py compare \
+              --threshold=5 \
```

Comment:
I understand that `sizes.py --help` explains the unit of `threshold`, but that helps very little when reading this file 🤷‍♂️
Not a blocker, just trying to explain my reasoning a bit further.

Reply:
I agree, but I also think that `0.05` is as ambigous as `5` if there's no unit :/

Reply:
Maybe - but `0.05` is less likely to be of unit _Bytes_ at least :)

Reply:
`--threshold-percent` would be very clear in any case

Reply:
We can do crimes instead:
```
comparison=$(
  python3 scripts/ci/sizes.py compare \
    --threshold=5% \
    "prev.json" "data.json"
)
```
The `%` is optional

Reply:
Haha, now we're talking 😆